.github/workflows/build-jax-wheels.yml

```diff
@@ -0,0 +1,86 @@
+name: Build Jax Wheels
+
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
+  pull_request:
+    # Check all PR
+
+#env:
+#  SPHERICART_NO_LOCAL_DEPS: "1"
+
+jobs:
+  build-jax-wheels:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["x86_64"]
+        python-version: ["3.10", "3.11", "3.12"]
+        cuda-version: ["12.4"]
+    env:
+      CIBW_SKIP: cp36-* cp37-* cp38-* cp39-*
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    
+    - name: Github Actions Envs Setup
+      run: |
+        CUVERSION="${{ matrix.cuda-version }}"
+        PYTHONVERSION="${{ matrix.python-version }}"
+
+        CU_VERSION_NO_DOT=${CUVERSION//./}
+        echo CU_VERSION_NO_DOT=${CU_VERSION_NO_DOT} >> $GITHUB_ENV
+
+        CU_VERSION_DASH=${CUVERSION//./-}
+        echo CU_VERSION_DASH=${CU_VERSION_DASH} >> $GITHUB_ENV
```

Comment:
For torch specifically, it depends if we want to support torch versions < 2.4.0, since the cu124 wheel only supports 2.4 as a minimum (https://download.pytorch.org/whl/cu124/torch). If we used cu121, we could support down to 2.1.0 (https://download.pytorch.org/whl/cu121/torch)?